apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/edit.js

```diff
@@ -34,11 +34,7 @@ const NavigationMenuEdit = ( { attributes } ) => {
 					/>
 				</Toolbar>
 			</BlockControls>
-			<ServerSideRender
-				attributes={ attributes }
-				block="a8c/navigation-menu"
-				className="wp-block-a8c-navigation-menu"
-			/>
+			<ServerSideRender attributes={ attributes } block="a8c/navigation-menu" />
```

Comment:
Because I moved the `wp-block-a8c-navigation-menu` class to the `<nav>` element to make the template part and front end display correctly the class is not needed at the block level now.